hcl_dx_jsf_demoportlet_archetype/src/main/resources/archetype-resources/store/logs/logger.log

```diff
@@ -0,0 +1,8013 @@
+2024-02-14 16:03:27 [deploy-portlet] [debug]: Deploy portlet execution started
```

Comment:
Do not think we want to check in the log file.

Reply:
I removed the log-file